include/swift/SIL/SILBuilder.h

```diff
@@ -1635,6 +1638,24 @@ class SILBuilder {
 
     BB->insert(InsertPt, TheInst);
   }
+
+  llvm::SmallString<16> getOperandTypeName(SILType OpdTy) {
+    auto &C = getASTContext();
```

Comment:
I don't think it is a good idea to return`llvm:: SmallString<16>` by value. May be pass an `out` as a by-reference parameter, i.e. `llvm::SmallString<16> &Name`?